neonvm/runner/main.go

```diff
@@ -579,12 +584,27 @@ func main() {
 	}
 
 	// leading slash is important
-	cgroupPath := fmt.Sprintf("/%s-vm-runner", vmStatus.PodName)
+	selfCgroupPath, err := getSelfCgroupPath()
+	if err != nil {
+		log.Fatalf("Failed to get self cgroup path: %s", err)
+	}
+	// Sometimes we'll get just '/' as our cgroup path. If that's the case, we should reset it so
+	// that the cgroup '/neonvm-qemu' still works.
+	if selfCgroupPath == "/" {
+		selfCgroupPath = ""
+	}
+	// ... but also we should have some uniqueness just in case, so we're not sharing a root level
+	// cgroup if that *is* what's happening. This *should* only be relevant for local clusters.
+	//
+	// We don't want to just use the VM spec's .status.PodName because during migrations that will
+	// be equal to the source pod, not this one, which may be... somewhat confusing.
+	cgroupPath := fmt.Sprintf("%s/neonvm-qemu-%s", selfCgroupPath, selfPodName)
```

Comment:
> our group could not use more resources than the parent cgroup. Is this correct?

That's correct. The limits from the parent cgroup will come from k8s container `resources.limits`, if there are any — so because VM pods aren't created with limits we should be good